### DISCUSSION TITLE
Support for other libraries

### DISCUSSION THREAD

**Discussion by q1b** [tailwindlabs/heroicons#636] [General]
## Heroicons are beautiful,
> its **build system** is also **very scalable** and **manageable** as well  as easy to extend support for other libraries

I just request for **support ** of more**libraries** that are likely, going to become more popular like **svelte** and **solidjs** as well,

I am also interested in making pull requests and contributions.

**Comment by carloslfu** [comment]
I like https://github.com/steeze-ui/icons by @JustinVoitel. It supports more frameworks, and it uses Heroicons 2.0.
